OvaryModelGenerator/Main.cxx

```diff
@@ -46,8 +60,37 @@ int main(int argc, char* argv[])
     
   delete pvsGen;
 
-  OvaryModelGenerator myOvary(nslices, nrot, d, h, w, outdirstr);
+  OvaryModelGenerator myOvary(nslices, nrot, d, h, w);
   myOvary.Generate();
+  auto ovarySections = myOvary.GetOutput();
```

Comment:
Moved scene export file to main. In future changes, it should also be modularized into some sort of exporter classes. For detailed reasons for this change, please go to the generator class for specific comments.